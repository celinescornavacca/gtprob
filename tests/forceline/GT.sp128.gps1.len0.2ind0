(((((QA0,((KD0,(AE0,BE0)),EC0)),DA0),BA0),((TA0,(YB0,ZB0)),(((RB0,SB0),GB0),BB0))),((((YD0,ZD0),((JB0,KB0),(LB0,(UB0,VB0)))),((((JA0,(AB0,(FC0,(OC0,PC0)))),(VA0,(KC0,(SE0,TE0)))),(((LD0,MD0),CB0),((PD0,QD0),NC0))),((((ZC0,AD0),UA0),((((GE0,HE0),XD0),(CE0,DE0)),QB0)),((NA0,((OB0,PB0),(XC0,YC0))),((WE0,XE0),SD0))))),(((AA0,(((((KE0,LE0),(RD0,(VD0,WD0))),IA0),(((((QE0,RE0),(ND0,OD0)),WA0),(MB0,(DC0,(OE0,PE0)))),((LC0,MC0),MA0))),((EB0,FB0),((KA0,LA0),(((((((UE0,VE0),DD0),(((WB0,XB0),ZA0),((IE0,JE0),NB0))),OA0),(PA0,((HC0,IC0),((GD0,HD0),DB0)))),(TD0,UD0)),(EE0,FE0)))))),((((FA0,(GA0,HA0)),(HB0,IB0)),((CC0,((BD0,CD0),UC0)),((RA0,((VC0,WC0),(JC0,(QC0,RC0)))),EA0))),((ME0,NE0),((TB0,(ED0,FD0)),(XA0,YA0))))),((ID0,JD0),((((AC0,BC0),SA0),(GC0,(SC0,TC0))),CA0)))));
