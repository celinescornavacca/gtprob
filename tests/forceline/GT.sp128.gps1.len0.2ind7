((((OA0,(WC0,XC0)),(((BA0,(((FC0,GC0),((QC0,RC0),(VA0,WA0))),(IA0,(QA0,(LB0,MB0))))),(((YA0,(KE0,LE0)),AA0),(ZA0,(JC0,(JD0,(PD0,QD0)))))),((KA0,((YC0,(KD0,LD0)),OB0)),CA0))),((PA0,((QB0,RB0),(RD0,SD0))),((((GE0,HE0),((WE0,XE0),((SE0,TE0),OD0))),((DA0,(((WB0,XB0),FB0),HA0)),(DC0,EC0))),((((((((ED0,FD0),NB0),(SC0,TC0)),((((IB0,(JB0,KB0)),NA0),LA0),JA0)),(((((HD0,ID0),GD0),SA0),(EA0,(((QE0,RE0),(TD0,(EE0,FE0))),BD0))),(((OE0,PE0),DE0),(FA0,((((((ZB0,AC0),PB0),(BB0,(MD0,ND0))),(WD0,(ME0,NE0))),((DB0,EB0),(HC0,IC0))),(((UC0,VC0),(RA0,((ZC0,AD0),XA0))),GA0)))))),((((BC0,CC0),(ZD0,AE0)),(((UE0,VE0),CB0),AB0)),(TA0,UA0))),((((BE0,CE0),(TB0,UB0)),(XD0,YD0)),(KC0,(IE0,JE0)))),(MA0,((PC0,(CD0,DD0)),HB0)))))),((((LC0,MC0),YB0),VB0),((UD0,VD0),(GB0,((NC0,OC0),SB0)))));
