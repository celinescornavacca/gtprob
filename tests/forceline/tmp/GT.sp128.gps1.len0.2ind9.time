2.81user 0.18system 0:03.28elapsed 91%CPU (0avgtext+0avgdata 163904maxresident)k
24inputs+8outputs (0major+26989minor)pagefaults 0swaps
