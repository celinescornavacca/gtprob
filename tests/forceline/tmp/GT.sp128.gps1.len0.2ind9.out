-162.41572034
